packages/lesswrong/components/walledGarden/PortalBarGcalEventItem.tsx

```diff
@@ -24,8 +25,10 @@ const styles = (theme) => ({
 
 const PortalBarGcalEventItem = ({classes, gcalEvent}) => {
   const { LWTooltip } = Components
+  const UrlClass = getUrlClass()
 
-  const urlParams = new URLSearchParams(gcalEvent.htmlLink.split('?')[1])
+  const url = new UrlClass(gcalEvent.htmlLink)
+  const urlParams = new URLSearchParams(url.search)
```

Comment:
@Discordius Unfortunately could not eliminate this line.

Reply:
This appears to just work fine for me.

Reply:
Yeah, this works (that's why it's in the PR :P). I was originally imagining that something like `url.SearchParams["eid"]` would be a valid property of the url object and eliminate the need for creating a URLSearchParams object.